distribution/scripts/jmeter/requests/102400B_buyStocks.xml

```diff
@@ -0,0 +1,996 @@
+<soapenv:Envelope xmlns:soapenv="http://schemas.xmlsoap.org/soap/envelope/">
```

Comment:
Format the xml payloads.

Reply:
Fixed in https://github.com/wso2/performance-ei/pull/9